### PR TITLE
[Enhance] Replace mmdet3d ops with mmcv ops

### PR DESCRIPTION
**Pull request by wHao-Wu**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Please describe the motivation of this PR and the goal you want to achieve through this PR.
Replace mmdet3d ops with mmcv ops.

## Modification
Please briefly describe what modification is made in this PR.
Use mmcv.ops instead of mmdet3d.ops in models.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mmdet3d/ops/__init__.py [hunk 1]
```diff
@@ -3,42 +3,25 @@
                       get_compiling_cuda_version, nms, roi_align,
                       sigmoid_focal_loss)
 
-from .ball_query import ball_query
+# from .ball_query import ball_query
```
Comment (Tai-Wang) [inline mmdet3d/ops/__init__.py:6]: Can remove it if it is not used?

**Review by Tai-Wang — COMMENTED** [review]
Just need to check this PR does not affect the performance and fix comments.
Please also fix the CI.

File: mmdet3d/ops/__init__.py (hunk 1 above)
Comment (wHao-Wu) [reply mmdet3d/ops/__init__.py:6]: Sorry. This line is just for debugging. I will remove this line in the next commit.

**Comment by ZCMax** [comment]
Please resolve the conflicts.

**Comment by wHao-Wu** [comment]
I want to leave the unittest here temporarily for checking the ops from mmcv.

File: tests/test_models/test_voxel_encoder/test_voxelize.py [hunk 2]
```diff
@@ -14,6 +14,8 @@ def _get_voxel_points_indices(points, coors, voxel):
 
 
 def test_voxelization():
+    if not torch.cuda.is_available():
+        pytest.skip('test requires GPU and torch+cuda')
```
Comment (Tai-Wang) [inline tests/test_models/test_voxel_encoder/test_voxelize.py:18]: Can we run this test with only cpu before? Does it use CPU-version voxelization?

Comment (wHao-Wu) [reply tests/test_models/test_voxel_encoder/test_voxelize.py:18]: Sure, we can use the CPU-version voxelization. But the CPU-version voxelization in mmcv has something wrong. The fixing processing of CPU-version voxelization is still in process. More details can be seen [here](https://github.com/open-mmlab/mmcv/pull/1746).

Comment (wHao-Wu) [reply tests/test_models/test_voxel_encoder/test_voxelize.py:18]: If we use the CPU-version voxelization now, this PR cannot pass the unittest. Or we can wait for the merging of the PR of fixing the CPU-version voxelization.

Comment (Tai-Wang) [reply tests/test_models/test_voxel_encoder/test_voxelize.py:18]: OK, we can temporarily use this judgment for now while also adding unit test for CPU-version afterward.

**Comment by ZwwWayne** [comment]
Need to resolve conflicts.

**Review by Tai-Wang — APPROVED** [review]

File: mmdet3d/ops/__init__.py [hunk 3]
```diff
@@ -3,42 +3,24 @@
                       get_compiling_cuda_version, nms, roi_align,
                       sigmoid_focal_loss)
 
-from .ball_query import ball_query
```
Comment (ZwwWayne) [inline mmdet3d/ops/__init__.py:6]: from mmcv.ops import ball_query

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**